### PR TITLE
Remove google search type field

### PR DESCRIPTION
**Pull request by bani**
I'm removing the Google search type field since it contains misleading information.

@CamDavidsonPilon @snormore 

(are you guys ok with the version bump and the trailing commas in the tests?)

### CODE REVIEW

**Comment by snormore** [comment]
👍

File: setup.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from distutils.core import setup
 
 setup(name='pyreferrer',
-      version='0.3.10.4',
+      version='0.3.10.5',
```
Comment (CamDavidsonPilon) [inline setup.py]: this is a major API change, I think we should bump to 3.11, maybe 4. Thoughts?

**Comment by CamDavidsonPilon** [comment]
@bani do you recall why we chose to do this? Still applicable?

**Comment by bani** [comment]
@CamDavidsonPilon yeah, Google is now using some http protocol thing which name I don't recall that sends only the domain in the referrer, not the path. So even for paid in most cases we don't have the `/aclk` visible anymore and therefore we'd be incorrectly categorizing as organic

**Comment by bani** [comment]
It's either "origin" or "strict origin", not sure: https://www.w3.org/TR/referrer-policy/#referrer-policy-origin

**Pull request closed without merging**